### DISCUSSION TITLE
Лайки и дизлайки

### DISCUSSION THREAD

**Discussion by aeifn** [PastVu/pastvu#333] [Ideas / Идеи и предложения]
В повседневном использовании сайта не хватает возможности плюсовать и минусовать комментарии (как на хабре), а также лайкать фотографии.

**Comment by paul-k-pastvu** [comment]
Не то, чтобы прямо не хватает ;)
А для какой, например, цели, это могло бы потребоваться? Чтобы что?

**Reply by aeifn** [reply]
Для того, чтобы высказать одобрение или благодраность комментатору, не пиша отдельный комментарий, который неизбежно будет низкокалориен и будет лишь захламлять дерево комментариев.

**Reply by paul-k-pastvu** [reply]
И этот же механизм использовать, что привлечь к комментарию внимание модератора.

**Comment by klimashkin** [comment]
Да, лайки и дизлайки это хорошая форма модераторства сообществом, там можно целую систему выстраивать.
У нас была дискуссия на эту тему некоторое время назад, пара моих комментариев:

> Ввести бюджет в единицах для каждого юзера. Отсыпать всем юзерам бюджет в сто единиц. Лайк их коммента: +1 к бюджету, дизлайк: -1. Когда на балансе меньше 50y.e., начать ограничивать rate, например не чаще одного коммента в пять минут. 0 и меньше - больше не может комментить.
> И раз в сутки отсыпать всем с отрицательным балансом по 10y.e чтобы они в итоге выбирались из ямы до следующего проишествия
> За опубликованное фото тоже отсыпать по 10ye

И так далее, можно много чего придумывать, давать звания за показатели и тд

**Reply by paul-k-pastvu** [reply]
Я, в принципе, уже по этому описанию готов возненавидеть такую систему (как пользователь) ;)

**Comment by paul-k-pastvu** [comment]
См. также #636
